.github/workflows/ci.yml

```diff
@@ -16,8 +16,14 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Install PHP
+        uses: shivammathur/setup-php@v1
```

Comment:
@sebastianbergmann 

If you decide to merge this, I will of course update #4153 - or vice versa.